include/dronecore.h

```diff
@@ -30,17 +59,83 @@ class DroneCore
         CONNECTIONS_EXHAUSTED
     };
 
+    /**
+     * Translates the ConnectionResult enum to a human-readable English string.
+     */
     static const char *connection_result_str(ConnectionResult);
 
+    /**
+     * Adds a UDP connection with the default arguments.
+     *
+     * This will listen on UDP port 14540 on localhost.
+     *
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection();
+
+    /**
+     * Adds a UDP connection with a specific port number.
+     *
+     * @param local_port_number The local UDP port to listen to
+     * @return the result of adding the connection.
+     */
     ConnectionResult add_udp_connection(int local_port_number);
 
+    /**
+     * Return vector of device UUIDs.
+     *
+     * This returns a vector of the UUIS of all devices that have been discovered so far and can be
+     * accessed.
+     *
+     * @return a reference to the vector containing the UUIDs
+     */
     const std::vector<uint64_t> &device_uuids() const;
+
+    /**
+     * Return the only discovered device.
```

Comment:
Probably more accurate to say "Return the first discovered device".

Reply:
Right 4f48319.